scripts/hail_batch/hgdp1kg_tobwgs_densify/hgdp_1kg_tob_wgs_densify.py

```diff
@@ -24,31 +24,19 @@ def query(output):  # pylint: disable=too-many-locals
 
     hl.init(default_reference='GRCh38')
 
-    # get TOB-WGS allele frequencies
+    hgdp_1kg = hl.read_matrix_table(GNOMAD_HGDP_1KG_MT)
     tob_wgs = hl.read_matrix_table(TOB_WGS).key_rows_by('locus', 'alleles')
-    tob_wgs = hl.experimental.densify(tob_wgs)
-    tob_wgs = tob_wgs.annotate_entries(GT=lgt_to_gt(tob_wgs.LGT, tob_wgs.LA))
-    tob_wgs = tob_wgs.annotate_rows(
-        gt_stats=hl.agg.call_stats(tob_wgs.GT, tob_wgs.alleles)
-    )
+    loadings = hl.read_table(GNOMAD_LIFTOVER_LOADINGS).key_by('locus', 'alleles')
 
-    # Get gnomAD allele frequency of variants that aren't in TOB-WGS
-    loadings_gnomad = hl.read_table(GNOMAD_LIFTOVER_LOADINGS).key_by('locus', 'alleles')
-    hgdp_1kg = hl.read_matrix_table(GNOMAD_HGDP_1KG_MT)
-    hgdp_1kg_row = hgdp_1kg.rows()[loadings_gnomad.locus, loadings_gnomad.alleles]
-    tob_wgs_row = tob_wgs.rows()[loadings_gnomad.locus, loadings_gnomad.alleles]
-    loadings_gnomad = loadings_gnomad.annotate(
-        gnomad_AF=hgdp_1kg_row.gnomad_freq.AF,
-        gnomad_popmax_AF=hgdp_1kg_row.gnomad_popmax.AF,
-        TOB_WGS_AF=tob_wgs_row.gt_stats.AF,
-    )
-    population_af_metadata = hgdp_1kg.gnomad_freq_meta.collect()
-    loadings_gnomad = loadings_gnomad.annotate_globals(
-        gnomad_freq_meta=population_af_metadata
+    # filter to loci that are contained in both tables and the loadings after densifying
+    tob_wgs = hl.experimental.densify(tob_wgs)
+    hgdp_1kg = hgdp_1kg.filter_rows(
+        hl.is_defined(loadings.index(hgdp_1kg['locus'], hgdp_1kg['alleles']))
+        & hl.is_defined(tob_wgs.index_rows(hgdp_1kg['locus'], hgdp_1kg['alleles']))
     )
-    gnomad_variants = loadings_gnomad.drop('loadings')
-    gnomad_variants_path = f'{output}/gnomad_annotated_variants_densified.ht'
-    gnomad_variants.write(gnomad_variants_path)
+    tob_wgs = tob_wgs.semi_join_rows(hgdp_1kg.rows())
+    tob_wgs_path = f'{output}/tob_wgs_filtered_densified.mt'
```

Comment:
There's already a `densify` in the path name.

```suggestion
    tob_wgs_path = f'{output}/tob_wgs_filtered.mt'
```